maven-core/src/main/java/org/apache/maven/utils/dag/Vertex.java

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
These classes seem to have simply been copied over from plexus. I'm no sure you can simply remove copyright like that but IANAL

Reply:
We can keep the codehaus header below the package I think.

Reply:
@gnodet 
Done.